dist/docker/Dockerfile

```diff
@@ -1,37 +1,47 @@
-
+# image for building
 FROM alpine:latest AS builder
 
-ARG BUILD_TYPE
-ARG RELEASE
-
-RUN if [ $RELEASE = "master" ] ; \
-    then \
-      wget https://github.com/qbittorrent/qBittorrent/archive/refs/heads/master.zip && \
-      unzip master.zip && \
-      cd qBittorrent-master ; \
-    else \
-      wget https://github.com/qbittorrent/qBittorrent/archive/refs/tags/release-${RELEASE}.tar.gz && \
-      tar xf release-${RELEASE}.tar.gz && \
-      cd qBittorrent-release-${RELEASE} ; \
-    fi && \
-    apk add --no-cache qt6-qttools-dev g++ libtorrent-rasterbar-dev cmake boost-dev ninja && \
-    cmake -B build-nox -G "Ninja" -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DGUI=OFF -DQT6=ON -DSTACKTRACE=OFF && \
-    cmake --build build-nox && \
-    cmake --build build-nox --target install/strip
-
+ARG QBT_VERSION
+
+# alpine linux qbittorrent package: https://git.alpinelinux.org/aports/tree/community/qbittorrent/APKBUILD
+
+RUN \
+  apk --update-cache add \
+    boost-dev \
+    cmake \
+    g++ \
+    libtorrent-rasterbar-dev \
+    ninja \
+    qt6-qtbase-dev \
+    qt6-qttools-dev && \
+  if [ "$QBT_VERSION" = "git" ]; then \
```

Comment:
I think I slightly preferred when `master` was used here, but I don't feel super strongly.

Reply:
Maybe `dev` or `devel`? Why to stick to git or something like that? IMO, it does not characterize the option properly.